src/main/scala/edu/ie3/simona/agent/grid/GridAgentBuilder.scala

```diff
@@ -51,13 +51,13 @@ import scala.jdk.OptionConverters.RichOptional
   *
   * @param gridAgentContext
   *   ActorContext of the grid agent
-  * @param environmentRefs
+  * @param _environmentRefs
   *   The environment refs
```

Comment:
Not sure what is my opinion here. Maybe @sebastian-peter has a good idea?

Reply:
With #1373 this class will be converted to an object. So my fix will only be used until #1373 is merged.

Reply:
Resolved!